app/src/main/java/com/eventyay/organizer/core/event/create/CreateEventViewModel.java

```diff
@@ -105,10 +105,28 @@ public boolean verify() {
             ZonedDateTime start = DateUtils.getDate(event.getStartsAt());
             ZonedDateTime end = DateUtils.getDate(event.getEndsAt());
 
-            if (!end.isAfter(start)) {
+            if (!end.isAfter(start) && start.getDayOfYear()==end.getDayOfYear() && start.getYear()==end.getYear()) {
```

Comment:
Since this is Zoned**DateTime**, this should work without any extra check, so please check ehy is it not working instead of adding extra checks